nautobot/extras/migrations/0021_customfield_changelog_data.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.admin.models import LogEntry
+from django.contrib.contenttypes.models import ContentType
+from django.db import migrations
+import uuid
+
+from nautobot.extras.choices import ObjectChangeActionChoices
+from nautobot.extras.models.customfields import CustomField
+from nautobot.extras.models.change_logging import ObjectChange
+from nautobot.utilities.utils import serialize_object
+
+
+def migrate_history(apps, schema_editor):
+    """
+    Migrate Django LogEntry objects to Nautobot ObjectChange objects.
+    """
+    custom_field_type = ContentType.objects.get_for_model(CustomField)
```

Comment:
In general, a migration should never directly import model classes, but should instead use `apps.get_model()` as needed

```suggestion
from django.db import migrations
import uuid

from nautobot.extras.choices import ObjectChangeActionChoices
from nautobot.utilities.utils import serialize_object


def migrate_history(apps, schema_editor):
    """
    Migrate Django LogEntry objects to Nautobot ObjectChange objects.
    """
    ContentType = apps.get_model("contenttypes", "ContentType")
    CustomField = apps.get_model("extras", "CustomField")
    LogEntry = apps.get_model("admin", "LogEntry")
    ObjectChange = apps.get_model("extras", "ObjectChange")
    custom_field_type = ContentType.objects.get_for_model(CustomField)
```